algoliasearch-java-net/pom.xml

```diff
@@ -10,6 +10,18 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>algoliasearch-java-net</artifactId>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>11</source>
+                    <target>11</target>
```

Comment:
Since Java 9, the `-release` option now supersedes both `-source` and `-target` and sets other flags correctly.

You can replace both `<source>` and `<target>` with `<release>11</release>`.

Reply:
Didn't know about it! thanks!
The `maven-compiler-plugin` seem to have this feature only starting from version 3.6 (we use 3.5.1), luckily, we can use plugin properties instead :D